File: app/src/main/java/com/example/capston/pcjari/GettingPHP.java
package com.example.capston.pcjari;

import android.os.AsyncTask;
import android.util.Log;

import org.json.JSONObject;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.net.HttpURLConnection;
import java.net.URL;

public class GettingPHP extends AsyncTask<String, Integer, String> {
    JSONObject jObject = null;

    @Override
    protected String doInBackground(String... params) {
        StringBuilder jsonHtml = new StringBuilder();
        try {
            URL phpUrl = new URL(params[0]);
            HttpURLConnection conn = (HttpURLConnection) phpUrl.openConnection();

            if (conn != null) {
                conn.setConnectTimeout(10000);
                conn.setUseCaches(false);

                if (conn.getResponseCode() == HttpURLConnection.HTTP_OK) {
                    BufferedReader br = new BufferedReader(new InputStreamReader(conn.getInputStream(), "UTF-8"));

                    while (true) {
                        String line = br.readLine();
                        if (line == null)
                            break;
                        jsonHtml.append(line + "\n");
                    }
                    br.close();
                }
                conn.disconnect();
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
        return jsonHtml.toString();
    }

    protected void onPostExecute(String str) {
        try {
            jObject = new JSONObject(str);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/com/example/capston/pcjari/Util/GettingPHP.kt
package com.example.capston.pcjari.Util

import android.os.AsyncTask
import android.util.Log
import com.example.capston.pcjari.Activity.A0_BaseActivity.Companion.TAG
import org.json.JSONObject
import java.io.BufferedReader
import java.io.InputStreamReader
import java.net.HttpURLConnection
import java.net.URL

class GettingPHP : AsyncTask<String?, Int?, String?>() {
    var jObject: JSONObject? = null
    override fun doInBackground(vararg params: String?): String? {
        val jsonHtml = StringBuilder()
        try {
            val phpUrl = URL(params[0])
            val conn = phpUrl.openConnection() as HttpURLConnection
            conn.connectTimeout = 10000
            conn.useCaches = false
            if (conn.responseCode == HttpURLConnection.HTTP_OK) {
                val br = BufferedReader(InputStreamReader(conn.inputStream, "UTF-8"))
                while (true) {
                    val line = br.readLine() ?: break
                    jsonHtml.append(line.trimIndent())
                }
                br.close()
            }
            conn.disconnect()
        } catch (e: Exception) {
            e.printStackTrace()
        }
        Log.d(TAG, "gettingPHP result : $jsonHtml")

        return jsonHtml.toString()
    }

    override fun onPostExecute(str: String?) {
        try {
            jObject = JSONObject(str)
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }
}